src/ECS/Systems/Systems.cpp

```diff
@@ -256,6 +256,51 @@ namespace Systems {
         }
     }
 
+    constexpr int outsideWindowTopLeft  = 0;
+    constexpr int outsideWindowBotRigth = 100;
+
+    static bool isOutsideWindow(const Types::Position &position)
```

Comment:
It would be nice to leave a small margin for each edge of the screen, so that if an entity goes halfway off the screen, it doesn't destroy it instantly.